pkgs/development/tools/misc/replacement/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, buildPythonApplication, isPy3k
+, ruamel_yaml
+, pytest, sh
+}:
+
+buildPythonApplication rec {
+  pname = "replacement";
+  version = "0.4.4";
+
+  disabled = !isPy3k;
```

Comment:
How about `disabled = isPy27`?

Reply:
Ok.

I've flattened all changes and force-pushed; I've added `Co-Authored-By: Doron Behar <doron.behar@gmail.com>` to the commit unless you'd rather not have it on there.

Thanks again for review and help improving this